homeassistant/components/roborock/strings.json

```diff
@@ -428,6 +428,24 @@
     "get_maps": {
       "name": "Get maps",
       "description": "Get the map and room information of your device."
+    },
+    "set_vacuum_goto_position": {
+      "name": "Go to position",
+      "description": "Send the vacuum to a specific position.",
+      "fields": {
+        "x_coord": {
```

Comment:
The input values use `x_coord` and `y_coord` and the output values use `x` and `y`. Can we pick a single name and use it for both input and output?

Reply:
I agree, we should change this to only use `x` and `y`

Reply:
https://github.com/home-assistant/core/pull/134133